Marlin/ubl_G29.cpp

```diff
@@ -1053,15 +1076,13 @@
       return UBL_ERR;
     }
 
-    /*
     if (code_seen('M')) {     // Check if a map type was specified
```

Comment:
Well...  That is a lot less cryptic than O.     That is how standards evolve.   There is push back against things that need to change.